templates/_components/static/list-events.twig

```diff
@@ -50,8 +52,8 @@
                         <div class="list-event__detail | paragraphs -xs | mb-2">
                             <p>
                                 Dr. Francisco will be the featured speaker at the
-                                annual men’s retreat on Sat March 16, for Our
-                                Shepherd Lutheran Church in Birmingham, MI.
+                                                                annual men’s retreat on Sat March 16, for Our
+                                                                Shepherd Lutheran Church in Birmingham, MI.
```

Comment:
@lindseybradford seems like there's some strange spacing/line breaks here and above.

Reply:
Some weirdness from VS extension conflicts. I think I worked out which ones weren't friendly over the weekend—at least I haven't seen this weird indentation happening lately.